content/logs/aws.md

```diff
@@ -0,0 +1,133 @@
+Enable the logging on your ELB first to collect your logs:
+{{< img src="logs/aws/configure_access_logs.png" alt="Configure Access Logs" >}}
+
+Set interval to **5 minutes** and define your s3 buckets:
+{{< img src="logs/aws/s3_location.png" alt="S3 Location" >}}
+
+Then go back to the Lambda function and define a trigger on the corresponding s3 bucket.
+
+#### CloudTrail
+
+AWS CloudTrail is an audit service. Use AWS CloudTrail to get a history of AWS API calls and related events for your account. This includes calls made by using the AWS Management Console, AWS SDKs, command line tools, and higher-level AWS services.
+
+You can identify which users and accounts called AWS for services that support CloudTrail, the source IP address from the calls, and when the calls occurred.
+You can integrate CloudTrail into applications using the API, automate trail creation for your Organization, check the status of your trails, and control how administrators turn CloudTrail logging on and off.
+
+When you define your Trails, you can select a s3 bucket to write the log in:
+{{< img src="logs/aws/tail_s3_selection.png" alt="S3 Selection" >}}
+
+You can send link the lambda function to this s3 bucket to send your logs.
+
+
+#### ECS
+
+ECS logs are the legacy Docker container. They are not directly related to the ECS service, but they correspond to the logs written by the running app (in the Docker Container). 
+
+Collect ECS logs directly from the containers thanks to our [Docker integration](/integrations/docker_daemon) (recommended).
+You can also [redirect those logs to Cloudwatch](http://docs.aws.amazon.com/AmazonECS/latest/developerguide/using_awslogs.html#w2ab1c21c21c13 ) and ask the lambda to ship them to your Datadog platform.
+
+#### CloudFront
+
+CloudFront is a CDN service which speeds up distribution of your static and dynamic web content, for example, .html, .css, .php and image files to end users.
+
+CloudFront delivers your content through a worldwide network of data centers called edge locations. When a user requests content that you're serving with CloudFront, the user is routed to the edge location that provides the lowest latency (time delay), so content is delivered with the best possible performance.
+
+
+When you enable logging for a distribution, specify the Amazon S3 bucket that you want CloudFront to store log files in. 
+If you're using Amazon S3 as your origin, we recommend that you do not use the same bucket for your log files; using a separate bucket simplifies maintenance.
+
+Store the log files for multiple distributions in the same bucket. When enabling logging, specify an optional prefix for the file names, to keep track of which log files are associated with which distributions, more information [here](http://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/AccessLogs.html#access-logs-choosing-s3-bucket ) 
+
+You can then collect the log from the s3 bucket thanks to the lambda function.
+
+
+
```

Comment:
We should add the same `what's next` section that we have after the Log Collection part (explore, processing, parsing)